data/speakers.ts

```diff
@@ -6042,6 +6042,26 @@ const speakers: Speaker[] = [
       city: 'New York',
     },
   },
+  {
+    objectID: 'gergelyke',
+    fullName: 'Gergely Nemeth',
+    tags: ['Node.js Security', 'Design Systems', 'Node.js', 'JavaScript'],
+    email: null,
+    website: 'https://nemethgergely.com/',
+    social: {
+      twitter: 'nthgergo',
+      github: 'gergelyke',
+      blog: null,
+      linkedin: 'https://www.linkedin.com/in/nthgergo/',
+    },
+    talks: [],
```

Comment:
@gergelyke Hi Gergely! Thank you very much for your PR! Just one little thing, could you please add your past/future talks? As an example please see `Sara Vieira` or `Fernando Migliorini Tenório`.